<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="c_noteButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>До</string>
    </property>
   </widget>
   <widget class="QPushButton" name="d_noteButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>41</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>Ре</string>
    </property>
   </widget>
   <widget class="QPushButton" name="e_noteButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>41</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>Ми</string>
    </property>
   </widget>
   <widget class="QPushButton" name="a_noteButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>41</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>Ля</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f_noteButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>41</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>Фа</string>
    </property>
   </widget>
   <widget class="QPushButton" name="g_noteButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>41</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>Соль</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_noteButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>41</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>Си</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>До#</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Ре#</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Фа#</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Соль#</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Ля#</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>307</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
